python/Ganga/Utility/feedback_report.py

```diff
@@ -652,6 +653,15 @@ def renameDataFiles(directory):
                 logger.debug("Err %s" % str(err))
                 writeErrorLog(str(sys.exc_info()[1]))
 
+        # Copy thread stack trace file
+        try:
+            thread_trace_source_path = os.path.join(getConfig('Configuration')['gangadir'], thread_trace_file_name)
+            thread_trace_target_path = os.path.join(fullLogDirName, thread_trace_file_name)
+            shutil.copyfile(thread_trace_source_path, thread_trace_target_path)
```

Comment:
Is there a reason we're copying the file from one destination to the other? Why not just write directly to the new location?